Add: Headless CMS user permission (#980)
vtexdocs/dev-portal-content: docs/faststore/docs/headless-cms/overview.mdx

## Before you begin

Before you proceed, please ensure the following:

<Steps>

> ### Access to a VTEX account.

Reviewer: ```suggestion
### Access to a VTEX account
```

Ensure you have a VTEX account to set up the Headless CMS with a FastStore project.

### Set up your FastStore project

If you do not have a FastStore project, refer to the [Setting up the project](https://developers.vtex.com/docs/guides/faststore/getting-started-2-setting-up-the-project) guide.

> 

Reviewer: ```suggestion
```

### Set up CMS resources with your VTEX user role

> You must have the following resources with your VTEX user role to be able to manage the Headless CMS:

Reviewer: ```suggestion
You must have the following [resources](https://help.vtex.com/en/tutorial/license-manager-resources--3q6ztrC8YynQf6rdc6euk3) associated with your VTEX Admin [user role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc) to be able to manage the Headless CMS:
```

| **Product** | **Category** | **Resource** |
| --------------- | ----------------- | ----------------- |
| CMS | cms | **See CMS menu on the top-bar** |
| CMS | cms | **Settings** |
| CMS | GraphQL | **CMS GraphQL API** |

> To associate a role with the resources above, you can [create a custom role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#creating-a-role) or edit an existing one.

Author: ```suggestion
To associate a role with the resources mentioned above, you can [create a custom role](https://help.vtex.com/en/tutorial/roles--7HKK5Uau2H6wxE1rH5oRbc#creating-a-role), add the resources to it, or add them to an existing role by editing it.
```
